src/app/about/page.tsx

```diff
@@ -1,7 +1,111 @@
 import Image from "next/image";
 import styles from "./About.module.css";
 
+import OfficerCard, { OfficerCardProperties } from "./officer-card";
+
+const officerCardData: OfficerCardProperties[] = [
+  {
+    position: "President",
+    image: "public/officers/Yonas_Bahre.jpg",
+    alt: "",
+    name: "Yonas Bahre",
+    socialLinksData: [
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/yonasbahre/",
+        icon: "/linkedin_logo.png",
+      },
+    ],
+  },
+  {
+    position: "Vice President",
+    image: "public/officers/Michael_Hayworth.jpg",
+    alt: "",
+    name: "Michael Hayworth",
+    socialLinksData: [
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/michaeldhayworth/",
+        icon: "/linkedin_logo.png",
+      },
+    ],
+  },
+  {
+    position: "Treasurer",
+    image: "public/officers/Stephen_Coomes.jpg",
+    alt: "",
+    name: "Stephen Coomes",
+    socialLinksData: [
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/stephen-coomes-8a4885221/",
+        icon: "/linkedin_logo.png",
+      },
+    ],
+  },
+  {
+    position: "Outreach Officer",
+    image: "public/officers/Angel_Lopez.png",
+    alt: "",
+    name: "Angel Lopez",
+    socialLinksData: [
+      {
+        name: "GitHub",
+        url: "https://github.com/angel1254mc",
+        icon: "/github_logo.svg",
+      },
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/angel1254/",
+        icon: "/linkedin_logo.png",
+      },
+      {
+        name: "Website",
+        url: "https://www.angel1254.com/",
+        icon: "/web_logo.svg",
+      },
+    ],
+  },
+  {
+    position: "Program Officer",
+    image: "public/officers/Param_Gupta.jpg",
+    alt: "",
+    name: "Param Gupta",
+    socialLinksData: [
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/paramg/",
+        icon: "/linkedin_logo.png",
+      },
+    ],
+  },
+  {
+    position: "Involvement Officer",
+    image: "public/officers/CJ_Weir.jpg",
+    alt: "",
+    name: "CJ Weir",
+    socialLinksData: [
+      {
+        name: "LinkedIn",
+        url: "https://www.linkedin.com/in/cj-weir/",
+        icon: "/linkedin_logo.png",
+      },
+    ],
+  },
+];
+
 export default function About() {
+  const officerCards = officerCardData.map((cardData) => (
+    <OfficerCard
+      position={cardData.position}
+      image={cardData.image}
+      alt={cardData.alt}
+      name={cardData.name}
+      socialLinksData={cardData.socialLinksData}
```

Comment:
[Suggestion]: You can use `{...cardData}` to spread props into the component, save a bit of work/duplication here.

Reply:
Can't change this due to eslint [react/jsx-props-no-spreading](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-props-no-spreading.md). I've decided to keep it as is rather than disable the rule. Let me know if you insist this be changed.

Reply:
That's fine, no strong opinion here. I'd guess that the use of TypeScript also helps catch many of the common bugs here.